src/Cli/dotnet/commands/dotnet-workload/install/NetSdkMsiInstallerClient.cs

```diff
@@ -186,7 +186,7 @@ public void GarbageCollect(Func<string, IWorkloadResolver> getResolverForWorkloa
             RemoveManifestsFromInstallStateFile(sdkFeatureBand);
 
         public new void SaveInstallStateManifestVersions(SdkFeatureBand sdkFeatureBand, Dictionary<string, string> manifestContents) =>
-            SaveInstallStateManifestVersions(sdkFeatureBand, manifestContents);
+            base.SaveInstallStateManifestVersions(sdkFeatureBand, manifestContents);
```

Comment:
Would it be simpler to make the base method public and delete this derived version?